ros2interface/ros2interface/verb/package.py

```diff
@@ -11,24 +11,52 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
+import collections
```

Comment:
You need whitespace both before and after this to pass flake8.

```suggestion

import collections

```